packages/@guardian/source-react-components-development-kitchen/src/components/divider/Divider.stories.tsx

```diff
@@ -7,7 +7,7 @@ import type { DividerProps } from './index';
 import { Divider } from './index';
 
 export default {
-	title: 'Kitchen/source-react-components-development-kitchen/Divider',
+	title: 'Source v4/source-react-components-development-kitchen/Divider',
```

Comment:
Maybe a pedantic point, but I wouldn't say the dev kitchen falls under "Source v4". I do see where you're coming from though 🤔  Will this make navigation simpler or content more discoverable?

Reply:
A tangential point. When I created the Foundations docs, I (probably mistakenly) did not organise them under a package. I think we can address this, either here or separately, by updating the title of Foundations docs from:

```tsx
<Meta
	title="Foundations/Colour"
	parameters={{ previewTabs: { canvas: { hidden: true } } }}
/>
```

```tsx
<Meta
	title="Source v4/source-foundations/Colour"
	parameters={{ previewTabs: { canvas: { hidden: true } } }}
/>
```
e.g. [colour.stories.mdx](https://raw.githubusercontent.com/guardian/source/main/packages/%40guardian/source-foundations/src/colour.stories.mdx)

Reply:
Agreed. I've dropped the commit that changes the kitchen stories. Let's raise a separate PR that updates foundations.